replica/backup.go

```diff
@@ -270,8 +275,12 @@ func preload(d *diffDisk) error {
 					length++
 				}
 			}
-			d.location[offset] = byte(i)
-			d.UsedBlocks++
+			d.rmLock.Lock()
+			if d.location[offset] < byte(i) {
+				d.location[offset] = byte(i)
+				d.UsedBlocks++
```

Comment:
is UsedBlocks coming correct now even with this change?

Reply:
We are not using `usedblock` in ci. `du -sh` is used to verify the sizes.

Reply:
ok.. so, to fix this still, we can do like
```
if d.location[offset] != byte(i) {
        d.UsedBlocks++
}
```
And, is there a way to know the value of this d.UsedBlocks during runtime?

Reply:
it seems - writeAt is also incrementing this..
so, do `d.UsedBlocks++` here, and we can do similar above check there during writes